lib/workers/repository/config-migration/branch/create.spec.ts

```diff
@@ -1,15 +1,11 @@
 import type { Indent } from 'detect-indent';
 import { Fixtures } from '../../../../../test/fixtures';
-import {
-  RenovateConfig,
-  getConfig,
-  partial,
-  platform,
-} from '../../../../../test/util';
-import { checkoutBranch, commitFiles } from '../../../../util/git';
+import { RenovateConfig, getConfig, partial } from '../../../../../test/util';
+import { scm } from '../../../../modules/platform/scm';
+import { checkoutBranch } from '../../../../util/git';
 import { createConfigMigrationBranch } from './create';
-import { MigratedDataFactory } from './migrated-data';
 import type { MigratedData } from './migrated-data';
+import { MigratedDataFactory } from './migrated-data';
```

Comment:
looks like your ide is doing some wrong import reordering

Reply:
What is wrong there? 
running "lint-fix" changes nothing (but this applies for the reverse ordering too).

Reply:
can you please revert the import reordering?